pipeline_trials/featurization_only.nf

```diff
@@ -0,0 +1,242 @@
+
+    output:
+    stdout emit: dummy_output_ch_txt
+
+    script:
+    """
+    cd ${baseDir}/../3.cellprofiling/slurm_scripts/ || exit 1
+    echo "Processing patient: ${patient}, well_fov: ${well_fov}, use_gpu: ${featurize_with_gpu}"
+    bash run_area_shape_child.sh ${patient} ${well_fov} TRUE ${featurize_with_gpu}
+    cd ${baseDir}/ || exit 1
+    """
+}
+
+process colocalization_cpu {
+    conda '/home/lippincm/miniforge3/envs/GFF_featurization'
+
+    tag { "colocalization_cpu" }
+
+    input:
+    tuple val(patient), val(well_fov), val(featurize_with_gpu)
+
+    output:
+    stdout emit: dummy_output_ch_txt
+
+    script:
+    """
+    cd ${baseDir}/../3.cellprofiling/slurm_scripts/ || exit 1
+    echo "Processing patient: ${patient}, well_fov: ${well_fov}"
+    bash run_colocalization_child.sh ${patient} FALSE ${well_fov}
+    cd ${baseDir}/ || exit 1
+    """
+}
+
+process colocalization_gpu {
+    conda '/home/lippincm/miniforge3/envs/GFF_featurization'
+
+    tag { "colocalization_gpu" }
+
+    input:
+    tuple val(patient), val(well_fov), val(featurize_with_gpu)
+
+    output:
+    stdout emit: dummy_output_ch_txt
+
+    script:
+    """
+    cd ${baseDir}/../3.cellprofiling/slurm_scripts/ || exit 1
+    echo "Processing patient: ${patient}, well_fov: ${well_fov}"
+    bash run_colocalization_child.sh ${patient} TRUE ${well_fov}
+    cd ${baseDir}/ || exit 1
+    """
+}
+
+process granularity_cpu {
+    conda '/home/lippincm/miniforge3/envs/GFF_featurization'
+
+    tag { "granularity_cpu" }
+
+    input:
+    tuple val(patient), val(well_fov), val(featurize_with_gpu)
+
+    output:
+    stdout emit: dummy_output_ch_txt
+
+    script:
+    """
+    cd ${baseDir}/../3.cellprofiling/slurm_scripts/ || exit 1
+    echo "Processing patient: ${patient}, well_fov: ${well_fov}"
+    bash run_granularity_child.sh ${patient} FALSE ${well_fov}
+    cd ${baseDir}/ || exit 1
+    """
+}
+
+process granularity_gpu {
+    conda '/home/lippincm/miniforge3/envs/GFF_featurization'
+
+    tag { "granularity_gpu" }
+
+    input:
+    tuple val(patient), val(well_fov), val(featurize_with_gpu)
+
+    output:
+    stdout emit: dummy_output_ch_txt
+
+    script:
+    """
+    cd ${baseDir}/../3.cellprofiling/slurm_scripts/ || exit 1
+    echo "Processing patient: ${patient}, well_fov: ${well_fov}"
+    bash run_granularity_child.sh ${patient} TRUE ${well_fov}
+    cd ${baseDir}/ || exit 1
+    """
+}
+
+process intensity_cpu {
+    conda '/home/lippincm/miniforge3/envs/GFF_featurization'
+
+    tag { "intensity_cpu" }
+
+    input:
+    tuple val(patient), val(well_fov), val(featurize_with_gpu)
+
+    output:
+    stdout emit: dummy_output_ch_txt
+
+    script:
+    """
+    cd ${baseDir}/../3.cellprofiling/slurm_scripts/ || exit 1
+    echo "Running GPU featurization for patient: ${patient}, well_fov: ${well_fov} use_gpu: ${featurize_with_gpu}"
+    bash run_intensity_child.sh ${well_fov} FALSE ${patient}
+    cd ${baseDir}/ || exit 1
```

Comment:
Consider adding comments to these blocks to help contextualize what these lines do.